roles/privacyidea/templates/privacyideaapp.py.j2

```diff
@@ -0,0 +1,23 @@
+import sys
+import socket
+sys.stdout = sys.stderr
+from privacyidea.app import create_app
+
+# Get the hostname
+hostname = socket.gethostname()
+
+if  'spshinfra' in hostname or 'dev' in hostname:
```

Comment:
in another j2 file it seems that using vars with {{}} was working, does it not work in this file?

Reply:
I did not use {{}} because those variables are used only once. I guess there is no need to initialize them as variables.

Reply:
i do not like the else if structure, I strongly preferer using vars. I am not sure why these vars are only used once, since i can not see that configure_privacyidea or the task that uses this field is executed only once. But even then i think that being able to control the over a var is the right way